openpype/plugins/publish/collect_audio.py

```diff
@@ -0,0 +1,83 @@
+import pyblish.api
+from pprint import pformat
+
+from openpype.client import (
+    get_last_version_by_subset_name,
+    get_representations,
+)
+from openpype.pipeline import (
+    legacy_io,
+    get_representation_path,
+)
+
+
+class CollectAudio(pyblish.api.InstancePlugin):
+    """ Collecting available audio subset to instance
+
+    """
```

Comment:
Can we improve this docstring? To me it doesn't really explain what this does?

Should this be more elaborate like:

```python
"""Collect asset's last published audio.

The audio subset name searched for is defined in:
    project settings > Collect Audio
"""
```